common/src/types/firestore-types.ts

```diff
@@ -47,8 +47,15 @@ export type FirestoreOneTimeTask = FirestoreCommonTask & {
   readonly type: 'ONE_TIME';
   readonly date: Date | firestore.Timestamp;
   readonly icalUID?: string;
+  readonly group?: string;
 };
```

Comment:
Do not nest this inside one time task. Create another type called `GroupTask`, and add that as a variant in `FirestoreTask`